### ISSUE TITLE
Trouble with docker run with 22.9.2 when os/arch has now changed to linux/arm64/v8

### ISSUE THREAD

**Issue by djowalker** [theopenconversationkit/tock-docker#28]
@vsct-jburet; @francoisno 

Running on ubuntu 22.0.4 LTS,
Docker version : `Docker version 20.10.12, build 20.10.12-0ubuntu2~20.04.1`
Docker-compose : `docker-compose version 1.25.0, build unknown`

There is a trouble when loading the tock/images.

When running for instance bot_admin we observe : `exec /bin/sh: exec format error`

Then running it with docker there is the following error.

```
docker run -entrypoint="/bin/bash" -i tock/bot_admin:22.9.2
WARNING: The requested image's platform (linux/arm64/v8) does not match the detected host platform (linux/amd64) and no specific platform was requested
exec /bin/sh: exec format error
```

I observed you changed the [OS/ARCH](https://hub.docker.com/r/tock/bot_admin/tags) with 22.9.2, is there a reason about that ?

**Comment by cd-mma** [comment]
I'm having the same problem.

**Comment by Matomatt** [comment]
The architecture changed from amd64 to arm64. We switched the os and everything worked fine afterwards.

**Comment by djowalker** [comment]
@Matomatt 
Ok it sounds interesting, you mean changing the os used by docker?
How do you do that ?

According to the docker version :
```
docker version
Client:
 Version:           20.10.12
 API version:       1.41
 Go version:        go1.16.2
 Git commit:        20.10.12-0ubuntu2~20.04.1
 Built:             Wed Apr  6 02:14:38 2022
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server:
 Engine:
  Version:          20.10.12
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.16.2
  Git commit:       20.10.12-0ubuntu2~20.04.1
  Built:            Thu Feb 10 15:03:35 2022
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.5.9-0ubuntu1~20.04.4
  GitCommit:        
 runc:
  Version:          1.1.0-0ubuntu1~20.04.1
  GitCommit:        
 docker-init:
  Version:          0.19.0
  GitCommit:       
```

**Comment by vsct-jburet** [comment]
This is because I've built locally the 22.9.2 with a Mac M2. Both amd & arm are now available for 22.9.2. Should fix the issue ;)
